docs/src/templates/pages/documentation/general/gulp.hbs

```diff
@@ -1,169 +1,120 @@
 
-	Open up a command prompt or terminal window and enter `npm install gulp -g`.  This should install Gulp so that Visual Studio can do it’s magic.
+3. **Install Yarn**
 
-3. **Install the project dependencies**
+    We currently use [Yarn](https://yarnpkg.com/en/) for our project dependencies.  Yarn is very similar to NPM with some small differences in performance and features.
 
-	This will eventually be done automatically by Visual Studio, but for the moment, we have to do it ourselves.
+    To install yarn check out the [current recommended process in their documentation](https://yarnpkg.com/en/docs/install) for your platform.
 
-	Navigate to the `/src/ConsumerWeb/` directory of your local ConsumerWeb project in a Command Prompt or Terminal window and then run `npm install`.  NPM will then install the project dependencies locally on your machine.  If you see any errors in your command prompt during this step, don’t worry about them, they shouldn’t affect the Gulp build.
+4. **Install the project dependencies**
 
-4. **Prioritise local Node config in VS**
-
-	Ok, so that’s the dependencies you need installed.  There is just one more option that needs adjusting within Visual Studio.
+    Navigate to the directory containing the projects `package.json` using your Command Prompt or Terminal window and then run `yarn`.  Yarn will then install the project dependencies locally on your machine.
 
-	In the Visual Studio Menu Bar, go to `Tools –> Options –> Projects and Solutions –> External Web Tools`.  Then, move the `$(PATH)` reference in the displayed list to the top of the list.  It should [look like this](http://i.imgur.com/SmzYdZs.png).
-
-…and that’s it!
-
-If you don’t plan on editing any CSS yourself, you can forget Gulp even exists and it’ll automatically do it’s thing when you build the project through Visual Studio.
-
-Otherwise, if you take a look at the Task Runner Explorer Window in Visual Studio, you will see all of the Gulp tasks available for you to run.
-
-If you want to get more familiar with the Gulp tasks in the project, [checkout what tasks have been pre-configured](#gulp-tasks) for use on the project.
-
-
-
-<hr class="sectionSplitter">
-
-<a name="gulp-cmd"></a>
-## Using Gulp on Command Line
-
-Another method of using Gulp is straight through the Command Line or Terminal.  If you’ve already followed the installation steps above, you should be able to do this by running `gulp` from the `/src/ConsumerWeb/` directory.
+You should now be able to run `gulp` from the directory containing the projects `package.json`.
 
 By default, Gulp will run the default task, but you can run any of the project tasks by appending the name of the task to your `gulp` call. For example, running`gulp css` will run the task that compiles the CSS files.
 
+For a full list of tasks avilable, [take a look at the modules documentation](https://github.com/justeat/gulp-build-fozzie#the-gulp-tasks).
```

Comment:
Alternatively, they can type `gulp --tasks` which will output a list of all available tasks to the command line.

Reply:
Have added in a line with this info in :smil